skipruntime-ts/core/src/internals/skipruntime_module.ts

```diff
@@ -1189,6 +1409,14 @@ class SkipRuntimeImpl implements SkipRuntime {
     }
   }
 
+  /**
+   * Subscribe to a reactive ressource according a given reactive response
+   * @param reactiveId - the reactive response collection
+   * @param since - the reactive response watermark
+   * @param f - the callback called on collection updates
+   * @param reactiveAuth The client user Skip session authentification corresponding to the reactive response
+   * @returns The subcription identifier
+   */
   subscribe<K extends TJSON, V extends TJSON>(
     reactiveId: string,
     since: Watermark,
```

Comment:
These two params are the unpacked "reactive response" referenced in the doc comment, right?

Why unpack them here?  Or, if it's better to unpack for some reason, why even bother with the `ReactiveResponse` type at all?

Reply:
Indeed, I asked myself the same question. No real reason to unpack.